### DISCUSSION TITLE
What does the "One Filesystem" checkbox do?

### DISCUSSION THREAD

**Discussion by vanasoft23** [cboxdoerfer/fsearch#425] [Q&A] [answered]
^

**Comment by cboxdoerfer** [comment, marked as answer]
This option means that FSearch will stay on the filesystem his folder belongs to when scanning for files. So when you include `/` and enable the "One Filesystem" option, then FSearch won't scan any sub-folder which belongs to a different filesystem (e.g. `/mnt/nas`).

**Reply by vanasoft23** [reply]
Alright that clears it up, thanks!
